<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visor de Simulaciones CFD (Dockable)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="vtkContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNueva_Simulacion"/>
    <addaction name="actionCargar_Simulacion"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar_Parametros"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuSimulacion">
    <property name="title">
     <string>Simulación</string>
    </property>
    <addaction name="actionEjecutar_Simulacion"/>
    <addaction name="actionDetener_Simulacion"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionDocumentacion"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionModo_Oscuro"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuSimulacion"/>
   <addaction name="menuVer"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="fileBrowserDock">
   <property name="windowTitle">
    <string>Navegador del Caso</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="fileBrowserTreeWidget">
       <column>
        <property name="text">
         <string>Archivos del Caso</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="parameterEditorDock">
   <property name="windowTitle">
    <string>Editor de Parámetros</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QScrollArea" name="parameterEditorScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>254</width>
          <height>138</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="parameterEditorLabel">
           <property name="text">
            <string>Seleccione un archivo para editar</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="parameterFormLayout"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logDock">
   <property name="windowTitle">
    <string>Log de Simulación</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPlainTextEdit" name="logPlainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNueva_Simulacion">
   <property name="text">
    <string>Nueva Simulación...</string>
   </property>
  </action>
  <action name="actionCargar_Simulacion">
   <property name="text">
    <string>Cargar Simulación...</string>
   </property>
  </action>
  <action name="actionGuardar_Parametros">
   <property name="text">
    <string>Guardar Parámetros</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionEjecutar_Simulacion">
   <property name="text">
    <string>Ejecutar Simulación</string>
   </property>
  </action>
  <action name="actionDetener_Simulacion">
   <property name="text">
    <string>Detener Simulación</string>
   </property>
  </action>
  <action name="actionDocumentacion">
   <property name="text">
    <string>Documentación</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionModo_Oscuro">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Modo Oscuro</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
